assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -690,6 +690,13 @@ che.factory.default_devfile_filenames=devfile.yaml,.devfile.yaml
 # which does not contain any {prod-short}-specific workspace descriptor.
 che.factory.default_editor=eclipse/che-theia/next
 
+# File size limit for the URL fetcher which fetch files from the SCM repository.
+che.factory.scm_file_fetcher_limit_bytes=102400
+
+# Additional files which may be present in repository to complement devfile v2,
+# and should be referenced as links to SCM resolver service in factory to retrieve them.
+che.factory.devfile2_files_resolution_list=devfile.yaml,.che/che-editor.yaml,.che/che-theia-plugins.yaml,.vscode/extensions.json
```

Comment:
how does it work together with `che.factory.default_devfile_filenames` ?

Reply:
good question, need to yhink that out (as a minimal solution is to add link to `.devfile.yaml` as well)

Reply:
Reworked to use devfile filename from factory